src/CommunityToolkit.Maui/Alerts/Snackbar/SnackBar.shared.cs

```diff
@@ -126,20 +126,26 @@ public async ValueTask DisposeAsync()
 	/// <summary>
 	/// Dispose Snackbar
 	/// </summary>
+#if ANDROID || IOS || MACCATALYST
 	protected virtual async ValueTask DisposeAsyncCore()
 	{
-#if ANDROID || IOS || MACCATALYST
-		if (nativeSnackbar is not null)
+		if (NativeSnackbar is not null)
```

Comment:
I just realized that if the GC doesn't run on the main thread, calling `NativeSnackbar` will throw an exception. Changing to `nativeSnackbar` bypasses the `MainThread.IsMainThread` check

```suggestion
		if (nativeSnackbar is not null)
```